synfig-studio/CMakeLists.txt

```diff
@@ -15,3 +16,35 @@ add_subdirectory(plugins)
 add_subdirectory(po)
 add_subdirectory(sounds)
 add_subdirectory(ui)
+
+file(MAKE_DIRECTORY "${SYNFIG_BUILD_ROOT}/share/appdata")
+file(MAKE_DIRECTORY "${SYNFIG_BUILD_ROOT}/share/applications")
+
+SYNFIG_INTLTOOL_MERGE(
+    TARGET_NAME appdata_xml
+    INPUT_FILE ${CMAKE_CURRENT_SOURCE_DIR}/org.synfig.SynfigStudio.appdata.xml.in
+    OUTPUT_FILE "org.synfig.SynfigStudio.appdata.xml"
+    INSTALL_DESTINATION "share/appdata"
+)
+
+SYNFIG_INTLTOOL_MERGE(
+    DESKTOP
+    TARGET_NAME desktop_file
+    INPUT_FILE ${CMAKE_CURRENT_SOURCE_DIR}/org.synfig.SynfigStudio.desktop.in
+    OUTPUT_FILE "org.synfig.SynfigStudio.desktop"
+    INSTALL_DESTINATION "share/applications"
+)
+
+add_custom_command(
```

Comment:
Better to do this part, like Inkscape did:
https://gitlab.com/inkscape/inkscape/-/blob/master/org.inkscape.Inkscape.desktop.template

Reply:
They used `configure_file` in a way we used it to configure `splash_screen_development.sif.in` where it replaces the `${INKSCAPE_ICONPATH}` with the value of that variable. Making changes to this would also require changes in autotools script, and I'm not too used with scripting in that.

Reply:
Agree.